openshift_images/using-imagestreams-with-kube-resources.adoc

```diff
@@ -8,4 +8,6 @@ toc::[]
 
 Image streams, being {product-title} native resources, work out of the box with all the rest of native resources available in {product-title}, such as builds or deployments. It is also possible to make them work with native Kubernetes resources, such as jobs, replication controllers, replica sets or Kubernetes deployments.
 
+include::snippets/default-namespaces.adoc[]
+
 include::modules/images-managing-images-enabling-imagestreams-kube.adoc[leveloffset=+1]
```

Comment:
I updated this to replace the note in the module with this snippet. As long as it's okay that that one specified any openshift-* or kube-* namespace (and our snippet is more narrow/specific).

I get what you mean that it might be a little nicer to be more specific to only just the feature we're talking about. But honestly it would be more of a maintenance nightmare if future updates need to be made and some people don't update in all places. Then the notices can get fragmented and potentially conflicting (as we've seen even in the files in this PR).

I'm going to leave it generic for now, but let me know if you feel strongly that we need to get more specific in some of these sections to only the feature we're talking about.

Reply:
+1, it is just a minor inconvenience and probably not worth the maintenance effort.

Thanks for updating the snippet. I think it should be more precise now. From looking at the code it should work in other `openshift-` namespaces, and I will test this just to be sure once I bring a cluster up.

Reply:
it is not simple: https://github.com/openshift/openshift-docs/pull/66132#discussion_r1358431179

Reply:
it is not simple: https://github.com/openshift/openshift-docs/pull/66132#discussion_r1358431179